app/scripts/views/change_password.js

```diff
@@ -10,12 +10,15 @@ define([
   'stache!templates/change_password',
   'views/mixins/password-mixin',
   'views/mixins/floating-placeholder-mixin',
+  'views/mixins/settings-mixin',
```

Comment:
Since `change_password` and `delete_account` are definitively [part of the settings](https://github.com/mozilla/fxa-content-server/pull/2748/files#diff-7dc4529b94b3a1ef3fd9ca92068a3bbeR131), both views should probably be moved into the `settings` subdirectory.